source/views/settings/icon.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+import React from 'react'
+import {ScrollView, Image, StyleSheet, Text} from 'react-native'
+import Icons from 'react-native-alternate-icons'
+import {Section, Cell} from 'react-native-tableview-simple'
+import {Column} from '../components/layout'
+import includes from 'lodash/includes'
+
+const styles = StyleSheet.create({
+  content: {
+    flex: 1,
+    paddingVertical: 10,
+  },
+  title: {
+    fontSize: 16,
+  },
+  icon: {
+    width: 16,
+    height: 16,
+    borderRadius: 5,
+    borderWidth: 0.5,
+  },
+})
+
+const icons = [
```

Comment:
I almost feel like this should be defined in /images, as we have other JS/image mappings there…

Reply:
Agreed